gammapy/maps/wcs.py

```diff
@@ -179,13 +179,15 @@ class WcsGeom(Geom):
         Reference pixel coordinate in each image plane.
     axes : list
         Axes for non-spatial dimensions
+    cutout_info : dict
+        Dict with cutout info, if the `WcsGeom` was created by `WcsGeom.cutout()`
     """
 
     _slice_spatial_axes = slice(0, 2)
     _slice_non_spatial_axes = slice(2, None)
     is_hpx = False
 
-    def __init__(self, wcs, npix, cdelt=None, crpix=None, axes=None):
+    def __init__(self, wcs, npix, cdelt=None, crpix=None, axes=None, cutout_info=None):
```

Comment:
Should `cutout_info` be an argument in the constructor? It could be added dynamically after the cutout is performed